client/src/modules/cohort/index.jsx

```diff
@@ -22,10 +20,10 @@ import FormDialog from 'common/components/FormDialog';
 import { archiveCohort, fetchCohortDetails } from './model/actions';
 import Dialog, { DialogContext } from 'common/components/Dialog';
 import ArchivedCohort from 'modules/cohort/components/ArchivedCohort';
-import GridList from 'common/components/GridList';
+import CollectionView from '../../common/components/CollectionView';
 import { ListType } from 'modules/list/consts';
 import MembersBox from 'common/components/Members';
-import { Routes } from 'common/constants/enums';
+import { ColorType, Routes } from 'common/constants/enums';
 import CohortHeader from './components/CohortHeader';
 import Preloader from '../../common/components/Preloader';
```

Comment:
change the path also here and check if there are any others `../../` routes like this across the project